lib/command/update/general_lastup_updater.rb

```diff
@@ -0,0 +1,106 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2013 whiteleaf. All rights reserved.
+#
+
+require_relative "../../narou/api"
+
+module Command
+  class Update < CommandBase
+    class GeneralLastupUpdater
+      def initialize(options)
+        @options = options
+        @database = Database.instance
+        @narou_novels = Hash.new { |h, k| h[k] = [] }
+        @other_novels = []
+        prepare
+      end
+
+      def save
+        @database.save_database
+      end
+
+      def prepare
+        @database.each.with_index do |(id, data), i|
```

Comment:
__[rubocop]__
* Lint/UnusedBlockArgument: Unused block argument - `data`. If it's necessary, use `_` or `_data` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Lint/UnusedBlockArgument: Unused block argument - `i`. If it's necessary, use `_` or `_i` as an argument name to indicate that it won't be used. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->